tests/nemo_text_processing/en/data_text_normalization/test_cases_punctuation.txt

```diff
@@ -1,18 +1,62 @@
 a `quote` example~a `quote` example
 a `25` example~a `twenty five` example
 single quote.'~single quote.'
-single quote. '4~single quote. 'four
+single quote. '4~single quote. ' four
 company's results,' said he:~company's results,' said he:
 test -~test -
 -5 test -~minus five test -
 abbreviation USA.~abbreviation USA.
 abbreviation U.S.A.~abbreviation USA
-abbreviation. U.S.A..~abbreviation. USA.
+abbreviation. U.S.A..~abbreviation. USA..
 1-4=5~one minus four equals five
-&hi;&hi;~and hi;and hi;
+&hi;&hi;~and hi; and hi;
 abbreviation USA,~abbreviation USA,
 23rd july, 1998~the twenty third of july, nineteen ninety eight
 April 29th’s meeting~april twenty ninth’s meeting
 ?,~?,
 ?,no~?,no
-I've 20' and 14/ they're I'm 16c.~I've twenty' and fourteen slash they're I'm sixteen c.
+I've 20' and 14/ they're I'm 16c.~I've twenty ' and fourteen slash they're I'm sixteen c.
+He's and I'm and Müller's and d'être~He's and I'm and Müller's and d'être
+here (4 June 2014). and~here (the fourth of june twenty fourteen). and
+``He's 50 and 2010``.~"He's fifty and twenty ten".
+"He's 50 and 2010".~"He's fifty and twenty ten".
+``test``.~"test".
+This (12 species) and...~This (twelve species) and...
+animals: ``cat``, ``dog``, ``lion``~animals: "cat", "dog", "lion"
+animals: (1), (2), (3)~animals: (one), (two), (three)
+animals: ``cat~animals: "cat
+animals: ``4~animals: " four
+non matching quotes?" "12text"~non matching quotes?" "twelve text"
+12"~twelve inches
+"12's and 3'sssssss"~"twelve's and three ' sssssss"
+This is ABC 'second ~This is ABC 'second
+``text 7`` text ``.~"text seven" text ".
+"text 7" text ".~"text seven inches text".
+"And now," he said, "He,~"And now," he said, "He,
+a: "b", "c", "d", "e", "f" or "g".~a: "b", "c", "d", "e", "f" or "g".
+a: "b" "c" "d" "e" "f" or "g".~a: "b" "c" "d" "e" "f" or "g".
+a: ``b``, ``c``, ``d``, ``e``, ``f`` or ``g``.~a: "b", "c", "d", "e", "f" or "g".
```

Comment:
this is duplicated with test_cases_punctuation_match_input.txt, can we only add things here, where punctuation does not match input? e.g. !1~! one

Reply:
and other punctuations